lib/Mojolicious/Guides/Cookbook.pod

```diff
@@ -452,7 +452,7 @@ nested closures that often result from continuation-passing style.
 
 =head2 Timers
 
-Another primary feature of the event loop are timers, which are created with
+Another primary feature of the event loop is timers, which are created with
```

Comment:
This is a tricky one, "timers" is a single feature but "timers" is plural. Perhaps better to use timers as the subject:
    Timers, another primary feature of the event loop, are created with ...